packages/contracts-bedrock/test/CommonTest.t.sol

```diff
@@ -125,14 +126,60 @@ contract SuperchainConfig_Initializer is CommonTest {
         proxy.upgradeToAndCall(
             address(SuperchainConfigImpl),
             abi.encodeCall(
-                SuperchainConfig.initialize, (systemOwner, initiator, vetoer, guardian, delay, maxPause, sequencers)
+                SuperchainConfig.initialize, (initiator, vetoer, guardian, operatingDelay, maxPause, sequencers)
             )
         );
 
         supConf = SuperchainConfig(address(proxy));
     }
 }
 
+contract DelayedVetoable_Init is SuperchainConfig_Initializer {
+    error Unauthorized(address expected, address actual);
+    error ForwardingEarly();
+
+    event Initiated(bytes32 indexed callHash, bytes data);
+    event Forwarded(bytes32 indexed callHash, bytes data);
+    event Vetoed(bytes32 indexed callHash, bytes data);
+
+    address target = makeAddr("target");
+    DelayedVetoable delayedVetoable;
+
+    function setUp() public override {
+        super.setUp();
+        delayedVetoable = new DelayedVetoable({
+            superchainConfig_: supConf,
```

Comment:
Why are these args postfixed instead of prefixed?

Reply:
Probably autosuggested tbh.
Will fix, but it's crazy that we don't have linting for this.

Reply:
OK, just for posterity, the reason for the postfix was that:
1. the internal getter function `_target` already had the prefix, so there was a naming collision.
2. I wanted to be consistent with the arg names on this function.

But I've resolved that differently by renaming it to `_targetContract`.